schema/testing.proto

```diff
@@ -55,123 +91,13 @@ service Testing {
   //
   // In cases where a test involves registering a final answer at the
   // end of the test, this method provides the means to do so.
-  rpc RegisterTest(RegisterTestRequest) returns (google.protobuf.Empty) {
+  rpc CheckTest(CheckTestRequest) returns (CheckTestResponse) {
```

Comment:
Preemptive apologies: I know `CheckTest` was my suggestion, but it feels a bit vague now. Perhaps `VerifyTest` would be better?